Build failure: ffmpeg-progress-yield

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

`nix build nixpkgs/master#python313Packages.ffmpeg-progress-yield`
or
`nix build nixpkgs/master#python312Packages.ffmpeg-progress-yield`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/305909042

### Relevant log output

```console
last 25 log lines:
       >     self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")
       >
       > ../../nix/store/ggk45c8i53zyrs8w9vy7l4bqmmsjjjlc-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/config/__init__.py:1474
       >   /nix/store/ggk45c8i53zyrs8w9vy7l4bqmmsjjjlc-python3.12-pytest-8.4.1/lib/python3.12/site-packages/_pytest/config/__init__.py:1474: PytestConfigWarning: Unknown config option: asyncio_mode
       >
       >     self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")
       >
       > test/test.py:153
       >   /build/source/test/test.py:153: PytestUnknownMarkWarning: Unknown pytest.mark.asyncio - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.asyncio
       >
       > test/test.py:164
       >   /build/source/test/test.py:164: PytestUnknownMarkWarning: Unknown pytest.mark.asyncio - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.asyncio
       >
       > test/test.py:178
       >   /build/source/test/test.py:178: PytestUnknownMarkWarning: Unknown pytest.mark.asyncio - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
       >     @pytest.mark.asyncio
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > FAILED test/test.py::TestAsyncLibrary::test_async_library - Failed: async def functions are not natively supported.
       > FAILED test/test.py::TestAsyncLibrary::test_async_quit - Failed: async def functions are not natively supported.
       > FAILED test/test.py::TestAsyncLibrary::test_async_quit_gracefully - Failed: async def functions are not natively supported.
       > ============ 3 failed, 7 passed, 2 deselected, 5 warnings in 36.16s ============
```

### Additional context

This can also be reproduced on `nixos-unstable-small` and `nixpkgs-unstable`.

### System metadata

    nix (Lix, like Nix) 2.93.3
    System type: x86_64-linux
    Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
    Features: gc, signed-caches
    System configuration file: /etc/nix/nix.conf
    User configuration files: /home/ulad/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/ulad/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/ulad/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/ulad/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
    Store directory: /nix/store
    State directory: /nix/var/nix
    Data directory: /nix/store/s7izz6akz2q6wilmbll41nrwnfrgfl68-lix-2.93.3/share

### Notify maintainers

@prusnak 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
+++ b/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
@@ -4,6 +4,7 @@
   fetchFromGitHub,
   setuptools,
   tqdm,
+  pytest-asyncio,
   pytestCheckHook,
   pythonOlder,
   ffmpeg,
@@ -31,14 +32,10 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    pytest-asyncio
     pytestCheckHook
     ffmpeg
     procps
-  ];
-
-  disabledTests = [
-    "test_quit"
-    "test_quit_gracefully"
   ];
 
   enabledTestPaths = [ "test/test.py" ];

--- a/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
+++ b/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
@@ -14,7 +14,7 @@
 buildPythonPackage rec {
   pname = "ffmpeg-progress-yield";
   version = "1.0.2";
-  format = "setuptools";
+  pyproject = true;
 
   disabled = pythonOlder "3.8";
 

--- a/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
+++ b/pkgs/development/python-modules/ffmpeg-progress-yield/default.nix
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "ffmpeg-progress-yield";
-  version = "0.11.3";
+  version = "1.0.2";
   format = "setuptools";
 
   disabled = pythonOlder "3.8";
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     owner = "slhck";
     repo = "ffmpeg-progress-yield";
     tag = "v${version}";
-    hash = "sha256-o5PlL6Ggo0Mrs/ujdnTV5GMAVeG2wpBoBDfxTVic3mA=";
+    hash = "sha256-tX4CioyhZvHNe5PItNwCF68ZQhs4fpG1ZrloGtei07I=";
   };
 
   build-system = [ setuptools ];

